tasks/ca/statcan/data.py

```diff
@@ -53,6 +53,22 @@ def download(self):
 
 
 class SplitAndTransposeData(BaseParams, Task):
+    INDIAN_RESERVES_SUFFIX = '-DQ'
+    DIVISION_COLUMN_NAMES = {
```

Comment:
- The division structure is inside the `SplitAndTranspose` to decouple the division from the `StatCanParser` as it only has to manage (parse, transpose) `CSV` files
- A tuple would be great but it's just another way of organizing the code. The regexp may be overkilling.
- I'm not sure if I get you... What would this new method do?